diary/views.py

```diff
@@ -39,11 +41,36 @@ def get(self, request, slug_field=None):
                 status_code = status.HTTP_409_CONFLICT
             result = {
                 'message': message,
-                # 'login_url': reverse('diary:login'),
+                'login_url': settings.SITE_ROOT + reverse('diary:login'),
             }
             return Response(result, status=status_code)
         except User.DoesNotExist:
             result = {
-                'message': "User account does not exist"
+                'detail': "User account does not exist"
             }
             return Response(result, status=status.HTTP_404_NOT_FOUND)
+
+
+class LoginUser(generics.GenericAPIView):
+
+    serializer_class = UserLoginSerializer
+
+    def post(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        credentials = {
+            'email': request.data['email'],
+            'password': request.data['password']
+        }
+        user = authenticate(request, **credentials)
+        if user:
+            jwt_payload_handler = api_settings.JWT_PAYLOAD_HANDLER
+            jwt_encode_handler = api_settings.JWT_ENCODE_HANDLER
+            payload = jwt_payload_handler(user)
+            token = jwt_encode_handler(payload)
+            response_data = {
+                'token': token,
+                'user': UserSignupSerializer(user, context={'request': request}).data
+            }
```

Comment:
LGTM, but I think it would be better if you abstract this implementation to the user Model.